akka-docs/src/test/java/jdocs/testkit/TestKitDocTest.java

```diff
@@ -162,7 +162,7 @@ public void demonstrateReceiveWhile() {
       getRef().tell(43, ActorRef.noSender());
       getRef().tell("hello", ActorRef.noSender());
 
-      final List<String> out = receiveWhile(duration("1 second"), in -> {
+      final List<String> out = receiveWhile(java.time.Duration.ofSeconds(1), in -> {
```

Comment:
can we remove the scala duration imports now?
```
 import scala.concurrent.duration.Duration;
 import scala.concurrent.duration.FiniteDuration;
```

Then I think `import java.time.Duration` should be added instead and use `Duration.ofSeconds` here, and in other places

Reply:
> can we remove the scala duration imports now?

Sure, I will remove the unused import.

> I think import java.time.Duration should be added instead and use Duration.ofSeconds

My original idea is it's used for documentation, so I used the full path for those. But I will change to `Duration.ofSeconds` at `OutputStreamSourceTest`.

Reply:
I understand, but we can't use full imports of all classes in docs, so java.time.Duration should become the "default" when it comes to Java API documentation.

Reply:
Got it. I will update those snippets.